client/blocks/post-edit-button/index.jsx

```diff
@@ -13,10 +13,10 @@ import { getEditURL } from 'lib/posts/utils';
 const PostEditButton = ( { post, site, size, onClick, translate } ) => {
 	const editUrl = getEditURL( post, site );
 	return (
-		<a className="post-edit-button" href={ editUrl } onClick={ onClick }>
+		<span className="post-edit-button" href={ editUrl } onClick={ onClick }>
```

Comment:
Not sure about the change to `span` here.

It's a straightforward link so `a` is the right choice semantically I think.